src/dlgTriggerEditor.cpp

```diff
@@ -397,7 +397,7 @@ dlgTriggerEditor::dlgTriggerEditor(Host* pH)
     exportAction->setEnabled(true);
     connect(exportAction, SIGNAL(triggered()), this, SLOT(slot_export()));
 
-    QAction* profileSaveAction = new QAction(QIcon(QStringLiteral(":/icons/document-save-all.png")), tr("Save Profile"), this);
+    profileSaveAction = new QAction(QIcon(QStringLiteral(":/icons/document-save-all.png")), tr("Save Profile"), this);
```

Comment:
Can we change this to `mProfileSaveAction` - it really helps *me* to keep track of which variables are members - and conversion of `QAction *`s from locals to members is something that will need to be done for all persistent/long-lasting `QAction`s so that they can be accessed and have their texts retranslated when the GUI Language is changed during a run... {The `mudlet` class is the other one that has a number of them do that to as well BTW! 🤓)